src/main/twirl/views/showOpportunity.scala.html

```diff
@@ -29,65 +29,132 @@
+                        </li>
+                    }
+                </ol>
+                <ol>
+                    @titleCols.map { case (col1, col2) =>
+                        <li>
+                            @col2.map { t => @Html(sectionLink(t._1, t._2)) }.getOrElse("")
+                        </li>
+                    }
+                </ol>
+            </nav>
+        </aside>
+
+
         <div>
-            <h2>Apply for funding</h2>
-            <a href="@controllers.routes.ApplicationController.showOrCreateForForm(applicationId)">Make an application</a>
-            <p></p>
-            <a href="@controllers.routes.OpportunityController.showGuidancePage(opportunity.id)">Guidance on seminars</a>
-            <p></p>
+            <h2 class="heading-large">Apply for funding</h2>
+
+            <p>
+                <a href="@controllers.routes.ApplicationController.showOrCreateForForm(applicationId)">Make an
+                    application</a>
+            </p>
+
+            <p>
+                <a href="@controllers.routes.OpportunityController.showGuidancePage(opportunity.id)">Guidance on
+                    seminars</a>
+            </p>
         </div>
 
         @currentSection.map { s =>
-            <article>
-                <h1>@s.sectionNumber. @s.title</h1>
-                @if(sectionNumber == 1) {
-                    <div>
-                        <h3>Start date</h3>
-                        @opportunity.startDate
-                        <h3>Days duration</h3>
-                        @opportunity.duration.map(d => s"${d.duration} ${d.units}").getOrElse("Open indefinitely")
-                        <h3>Opportunity value</h3>
-                            &pound; @opportunity.value.amount @opportunity.value.unit
-                    </div>
-                    <p></p>
-                }
-                @s.paragraphs.map { p =>
-                    <p>@p</p>
-                }
-            </article>
-            <footer>
+
+            @if(sectionNumber == 1) {
+
+        <div class="panel panel-border-narrow">
+            <aside role="complementary" class="notice">
+                <div class="column-half">
+                    <dl>
+                        <dt class="heading-small">Deadline</dt>
+                        <dd>Apply any time</dd>
+                    </dl>
+                </div>
+
+                <div class="column-half">
+                    <dl>
+                        <dt class="heading-small">Value</dt>
+                        <dd>&pound;@opportunity.value.amount @opportunity.value.unit</dd>
+                    </dl>
+                </div>
+            </aside>
+        </div>
+
+            }
+        <article>
+            <h2 class="heading-large">@s.sectionNumber. @s.title</h2>
+            @s.paragraphs.map { p =>
+                <p class="text">@p</p>
+            }
+        </article>
+
+        <footer class="contain-floats">
```

Comment:
In the design, this is on the full width of the page, so I'd put this in its own
```

<div class="grid-row"> 
  <div class="column-full">
    ...
  </div>
</div>
```

HAVING SAID THAT, I prefer it being part of the 2/3rds width, but maybe worth checking with design?